src/components/ErrorModal/ErrorModal.js

```diff
@@ -37,12 +37,14 @@ function ErrorModal(props) {
   };
 
   const errorsToDisplay = [];
+  const itemLimitKey = 'itemLimit';
```

Comment:
Could you please rename itemLimitKey  to ITEM_LIMIT_KEY?

Reply:
Fixed